src/main/java/ru/sbt/mipt/event/handler/DoorCloseEvenHandler.java

```diff
@@ -0,0 +1,22 @@
+package ru.sbt.mipt.event.handler;
+
+import ru.sbt.mipt.command.TurnOffAllLightsCommand;
+import ru.sbt.mipt.event.DoorCloseEvent;
+
+public class DoorCloseEvenHandler {
+    final private DoorCloseEvent event;
+
+    public DoorCloseEvenHandler(DoorCloseEvent event) {
+        this.event = event;
+    }
+
+    public void handle() {
+        event.getDevice().setOpen(false);
+        System.out.println("Door " + event.getDevice().getId() + " in room " + event.getPlace().getName() + " was closed.");
+
+        if (event.getPlace().getName().equals("hall")) {
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D1%80%D0%B0%D0%B7%D0%B4%D0%B5%D0%BB%D1%8F%D0%B9%D1%82%D0%B5-%D0%B8%D0%B7%D0%BC%D0%B5%D0%BD%D0%B5%D0%BD%D0%B8%D0%B5-%D0%BE%D0%B1%D1%8A%D0%B5%D0%BA%D1%82%D0%BE%D0%B2-%D0%B8-%D1%81%D1%86%D0%B5%D0%BD%D0%B0%D1%80%D0%B8%D0%B8-%D1%83%D0%BC%D0%BD%D0%BE%D0%B3%D0%BE-%D0%B4%D0%BE%D0%BC%D0%B0